### PR TITLE
#601

### PR DESCRIPTION
**Pull request by vithanichandresh**
Qr code issue solved

### CODE REVIEW

File: android/app/google-services.json [hunk 1]
```diff
@@ -1,40 +1,72 @@
 {
   "project_info": {
-    "project_number": "347734179578",
-    "project_id": "app-2i2i",
-    "storage_bucket": "app-2i2i.appspot.com"
+    "project_number": "453884442411",
+    "project_id": "i2i-test",
+    "storage_bucket": "i2i-test.appspot.com"
```
Comment (2i2i) [inline android/app/google-services.json:5]: why test? for the prod app, we should not have anything with test

File: android/app/src/main/google-services.json [hunk 2]
```diff
@@ -1,40 +1,72 @@
 {
   "project_info": {
-    "project_number": "347734179578",
-    "project_id": "app-2i2i",
-    "storage_bucket": "app-2i2i.appspot.com"
+    "project_number": "453884442411",
+    "project_id": "i2i-test",
```
Comment (2i2i) [inline android/app/src/main/google-services.json:4]: test?

File: android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -60,20 +60,38 @@
...
             </intent-filter>
 
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data
+                    android:host="test.2i2i.app"
```
Comment (2i2i) [inline android/app/src/main/AndroidManifest.xml:80]: test?

File: lib/infrastructure/models/user_model.dart [hunk 4]
```diff
@@ -113,6 +113,7 @@ class UserModel extends Equatable {
 
   UserModel({
     required this.id,
+    this.url = '',
```
Comment (2i2i) [inline lib/infrastructure/models/user_model.dart:116]: are you changing the db model?

File: android/app/google-services.json (hunk 1 above)
Comment (Androidsignal) [reply android/app/google-services.json:5]: This is just latest json from firebase.

File: android/app/src/main/google-services.json (hunk 2 above)
Comment (Androidsignal) [reply android/app/src/main/google-services.json:4]: This is just latest json from firebase.

File: android/app/google-services.json (hunk 1 above)
Comment (Androidsignal) [reply android/app/google-services.json]: This has been fixed.

File: android/app/src/main/google-services.json (hunk 2 above)
Comment (Androidsignal) [reply android/app/src/main/google-services.json]: Fixed.

File: android/app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (Androidsignal) [reply android/app/src/main/AndroidManifest.xml]: removed.

File: lib/infrastructure/models/user_model.dart (hunk 4 above)
Comment (Androidsignal) [reply lib/infrastructure/models/user_model.dart:116]: Yes, we need to save the deep link URL that's why.

**Pull request merged**